### PR TITLE
feature: added new classes

### PR DESCRIPTION
**Pull request by akalinbet**

### CODE REVIEW

File: src/com/itea/homework03/flowers/FlowerStore.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            if (i < countOfTulip) {
+                bouquet.add(new Tulip());
+            }
+        }
+        countMoney(bouquet);
+        return bouquet;
+    }
+
+    private void countMoney(List<Flower> flowers) {
+        for (Flower flower : flowers) {
```
Comment (Aleksandr04) [inline src/com/itea/homework03/flowers/FlowerStore.java]: please transform it to Stream

File: src/com/itea/homework03/flowers/FlowersSaver.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import java.io.Writer;
+import java.util.List;
+
+public final class FlowersSaver {
+    private FlowersSaver() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static void save(String path, List<Flower> flowers) {
+        try (Writer writer = new FileWriter(path);
```
Comment (Aleksandr04) [inline src/com/itea/homework03/flowers/FlowersSaver.java]: Please don't do this.. Only a single line should be present inside try()

File: src/com/itea/homework03/flowers/FlowersSaver.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+             BufferedWriter bufferedWriter = new BufferedWriter(writer)){
+
+            bufferedWriter.write(flowers.toString());
+            bufferedWriter.flush();
+
+        } catch (IOException e){
+            e.printStackTrace();
```
Comment (Aleksandr04) [inline src/com/itea/homework03/flowers/FlowersSaver.java]: change it on sout("error message");

File: src/com/itea/homework03/flowers/Flower.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.itea.homework03.flowers;
+
+public class Flower {
+
+    private double price;
+
+    public Flower() {
```
Comment (Aleksandr04) [inline src/com/itea/homework03/flowers/Flower.java]: can you remove empty constructor?

Comment (akalinbet) [reply src/com/itea/homework03/flowers/Flower.java]: no, because I have constructor with price and without price. Without price used in classes instances

**Pull request merged**